doc/examples.md

```diff
@@ -22,7 +78,38 @@ public class SchemaImport {
 }
 ```
 
-**Nf Core pipeline output**
+To validate the Nanopore instrument output, the schema can be loaded by the [Everit](https://github.com/everit-org/json-schema)
+`SchemaLoader` class:
+
+```java
+InputStream schemaStream = OxfordNanoporeInstrumentOutput.getSchemaAsStream()
+JSONObject rawSchema = new JSONObject(new JSONTokener(schemaStream))
+Schema jsonSchema = SchemaLoader.load(rawSchema)
+```
+
```

Comment:
How is the validation relevant for the usage of the object. I think it would be sufficient to point to how the validation can be done with the core utils lib